src/components/leftCol/YourAccs.vue

```diff
@@ -4,9 +4,14 @@
             class="card custom-card is-flex-direction-column is-align-items-center is-justify-content-space-between custom-card-padding"
           >
             <h2 class="custom-card-title">vos comptes</h2>
+            <template v-if="moneyAccounts.length > 0">
             <Acc v-for="account in moneyAccounts" :bal="account.bal" :curr="account.curr">
               <template v-slot:name>{{ account.name.split(':')[0] }}</template>
             </Acc>
+            </template>
+            <template v-else>
+              <p class="notification is-default">Vous n'avez pas encore de compte</p>
+            </template>
```

Comment:
This will be displayed while charging... and I feel this can be distressing. The best would be to have a placeholder while charging, and finish on that text if accounts.length is 0 and we know it.

Reply:
Yes.
Did you create the issue #36 for this ?

Reply:
yup, I pushed all this PR and removed this message for now while loading. So #36 is still needed, as placeholder is better than nothing. I had to take a decision and felt that no message was better than one while loading. Please note this message still displays itself when all accounts finished loading and we find our selves with... no accounts.